assets/state-worker.yaml

```diff
@@ -0,0 +1,196 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nfd-worker
+  namespace: openshift-nfd
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: nfd-worker
+  namespace: openshift-nfd
+rules:
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+  resourceNames:
+  - nfd-worker
+- apiGroups:
+  - security.openshift.io
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+  resourceNames:
+  - nfd-worker
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: nfd-worker
+  namespace: openshift-nfd
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nfd-worker
+subjects:
+- kind: ServiceAccount
+  name: nfd-worker
+  namespace: openshift-nfd
+userNames:
+- system:serviceaccount:openshift-nfd:nfd-worker
+--- 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nfd-worker
+  namespace: openshift-nfd
+data:
+  nfd-worker-conf: |
+    sources:
+      pci:
+        deviceLabelFields:
+          - "vendor"
+        deviceClassWhitelist:
+          - "0200"
+          - "03"
+          - "12"
+    
+    #      - "class"
```

Comment:
Why is this and the following 3 lines commented out?

Reply:
This is the config file, left it as a hint how the config file looks like.